doc/introduction/index.rst

```diff
@@ -3,383 +3,363 @@
+and virtualization solutions can support device passthrough (using VT-d),
+including Xen, KVM, and ACRN hypervisor.  In most cases, the guest operating
+system in the User VM, must be compiled to support passthrough by using kernel
+build-time options.
+
+
+Static Configuration Based on Scenarios
+***************************************
+
+Scenarios are a way to describe the system configuration settings of the ACRN
+hypervisor, VMs, and resources they have access to that meet your specific
+application's needs such as compute, memory, storage, graphics, networking, and
+other devices.  Scenario configurations are stored in an XML format file and
+edited using the ACRN configurator tool.
+
+Following a general embedded-system programming model, the ACRN hypervisor is
+designed to be statically customized at build time per hardware and scenario,
+rather than providing one binary for all scenarios.  Dynamic configuration
+parsing is not used in the ACRN hypervisor for these reasons:
+
+* **Reduce complexity**. ACRN is a lightweight reference hypervisor, built for
+  embedded IoT. As new platforms for embedded systems are rapidly introduced,
+  support for one binary could require more and more complexity in the
+  hypervisor, which is something we strive to avoid.
+* **Maintain small footprint**. Implementing dynamic parsing introduces hundreds or
+  thousands of lines of code. Avoiding dynamic parsing helps keep the
+  hypervisor's Lines of Code (LOC) in a desirable range (less than 40K).
+* **Improve boot time**. Dynamic parsing at runtime increases the boot time. Using a
+  static build-time configuration and not dynamic parsing helps improve the boot
+  time of the hypervisor.
+
+The scenario XML file together with a target HW board XML file are used to build
+the ACRN hypervisor image tailored to your HW and application needs. The ACRN
+project provides a board inspector tool to automatically create the board XML
+file by inspecting the target hardware. ACRN also provides a
+:ref:`configurator tool <acrn_configuration_tool>`
+to create and edit a tailored scenario XML file based on pre-defined sample
+scenario configurations.
 
-   Hybrid RT scenario
+.. _usage-scenarios:
 
-Logical Partition Scenario
-==========================
+Pre-Defined Sample Scenarios
+****************************
 
-This scenario is a simplified configuration for VM logical
-partitioning: both User VMs are independent and isolated, they do not share
-resources, and both are automatically launched at boot time by the hypervisor.
-The User VMs can be Real-Time VMs (RTVMs), Safety VMs, or standard User VMs.
+Project ACRN provides some pre-defined sample scenarios to illustrate how you
+can define your own configuration scenarios.
 
-.. figure:: images/ACRN-Logical-Partition.png
-   :width: 600px
-   :align: center
-   :name: logical-partition
 
-   Logical Partitioning scenario
+* **Industry** is a traditional computing, memory, and device resource sharing
```

Comment:
```suggestion
* **Shared** is a traditional computing, memory, and device resource sharing
```

Reply:
Alas, it's still called "Industry" in 2.6 but will be called "shared" in 2.7